src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -44,46 +43,127 @@ private class ConfigurationCacheItem
+        /// </summary>
         public ITargetingContextAccessor TargetingContextAccessor { get; init; }
 
+        /// <summary>
+        /// Options controlling the targeting behavior for feature variant allocation
+        /// </summary>
+        public TargetingEvaluationOptions AssignerOptions
+        {
+            get => _assignerOptions;
+
+            init
+            {
+                _assignerOptions = value ?? throw new ArgumentNullException(nameof(value));
+            }
+        }
+
+        /// <summary>
+        /// Checks whether a given feature is enabled.
+        /// </summary>
+        /// <param name="feature">The name of the feature to check.</param>
+        /// <returns>True if the feature is enabled, otherwise false.</returns>
         public Task<bool> IsEnabledAsync(string feature)
         {
             return IsEnabledWithVariantsAsync<object>(feature, appContext: null, useAppContext: false, CancellationToken.None).AsTask();
         }
 
+        /// <summary>
+        /// Checks whether a given feature is enabled.
+        /// </summary>
+        /// <param name="feature">The name of the feature to check.</param>
+        /// <param name="appContext">A context providing information that can be used to evaluate whether a feature should be on or off.</param>
+        /// <returns>True if the feature is enabled, otherwise false.</returns>
         public Task<bool> IsEnabledAsync<TContext>(string feature, TContext appContext)
         {
             return IsEnabledWithVariantsAsync(feature, appContext, useAppContext: true, CancellationToken.None).AsTask();
         }
 
+        /// <summary>
+        /// Checks whether a given feature is enabled.
+        /// </summary>
+        /// <param name="feature">The name of the feature to check.</param>
+        /// <param name="cancellationToken">The cancellation token to cancel the operation.</param>
+        /// <returns>True if the feature is enabled, otherwise false.</returns>
         public ValueTask<bool> IsEnabledAsync(string feature, CancellationToken cancellationToken)
         {
             return IsEnabledWithVariantsAsync<object>(feature, appContext: null, useAppContext: false, cancellationToken);
         }
 
+        /// <summary>
+        /// Checks whether a given feature is enabled.
+        /// </summary>
+        /// <param name="feature">The name of the feature to check.</param>
+        /// <param name="appContext">A context providing information that can be used to evaluate whether a feature should be on or off.</param>
+        /// <param name="cancellationToken">The cancellation token to cancel the operation.</param>
+        /// <returns>True if the feature is enabled, otherwise false.</returns>
         public ValueTask<bool> IsEnabledAsync<TContext>(string feature, TContext appContext, CancellationToken cancellationToken)
         {
             return IsEnabledWithVariantsAsync(feature, appContext, useAppContext: true, cancellationToken);
```

Comment:
The public methods should be before all the private methods. Looks like a private method got snuck in with the publics.